sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -973,6 +972,9 @@ export class MessageReceiver extends LinkEntity {
         );
         try {
           this._onError(err);
+          // Once the user's error handler has been called,
+          // close the receiver to prevent future messages/errors from being received.
+          await this.close();
```

Comment:
Also, does any error from _this_ close() need to get funneled to the user's error handler?

Reply:
I would say __no__ if we're trying to state that their error handler shouldn't be called with an error more than once (though that's already broken if message locks are being renewed).

Reply:
Makes sense to me. :)